runtimes/eden/src/xcm_config.rs

```diff
@@ -151,3 +162,76 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+pub struct AccountIdToMultiLocation;
+impl Convert<AccountId, MultiLocation> for AccountIdToMultiLocation {
+	fn convert(account: AccountId) -> MultiLocation {
+		X1(AccountId32 {
+			network: NetworkId::Any,
+			id: account.into(),
+		})
+		.into()
+	}
+}
+parameter_types! {
+	pub const BaseXcmWeight: XcmWeight = 100_000_000; // TODO: recheck this
+	pub const MaxAssetsForTransfer: usize = 2;
+}
+parameter_types! {
+	pub SelfLocation: MultiLocation = MultiLocation::here();
+}
+parameter_type_with_key! {
+	pub ParachainMinFee: |_location: MultiLocation| -> Option<u128> {
+		Some(u128::MAX)
+	};
+}
+#[derive(Encode, Decode, Eq, PartialEq, Clone, PartialOrd, Ord, TypeInfo, RuntimeDebug)]
+pub enum CurrencyId {
+	// NODL native token
+	NodleNative,
+}
+pub struct CurrencyIdConvert;
+impl Convert<CurrencyId, Option<MultiLocation>> for CurrencyIdConvert {
+	fn convert(id: CurrencyId) -> Option<MultiLocation> {
+		match id {
+			CurrencyId::NodleNative => Some(NodlLocation::get()),
+		}
+	}
+}
+impl Convert<MultiLocation, Option<CurrencyId>> for CurrencyIdConvert {
+	fn convert(location: MultiLocation) -> Option<CurrencyId> {
+		if location == MultiLocation::parent() {
+			return None;
+		}
+		match location.interior() {
+			&Junctions::X2(Parachain(para_id), PalletInstance(key)) => {
+				if (
+					u32::from(ParachainInfo::parachain_id()),
+					<Balances as PalletInfoAccess>::index() as u8,
+				) == (para_id, key)
+				{
+					Some(CurrencyId::NodleNative)
+				} else {
+					None
+				}
+			}
+			_ => None,
+		}
+	}
+}
+
+impl orml_xtokens::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Balance = Balance;
+	type CurrencyId = CurrencyId;
+	type CurrencyIdConvert = CurrencyIdConvert;
+	type AccountIdToMultiLocation = AccountIdToMultiLocation;
+	type SelfLocation = SelfLocation;
+	type XcmExecutor = XcmExecutor<XcmConfig>;
+	type Weigher = FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
```

Comment:
For this sake of this line, I checked `UnitWeightCost` and noticed its value seems too large comparing to the BaseXcmWeight. Can you please readjust that to something like `200_000_000`?

Reply:
done